### PR TITLE
Test/schema coverage

### PR DESCRIPTION
**Pull request by datbth**
Track schema coverage in tests (see https://github.com/interagent/committee/issues/296)

## Other changes
* https://github.com/interagent/committee/pull/297/commits/32600b24c4835e138a72b376f62885b8fbcf0bf5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -403,6 +403,38 @@ end
 
 The default assertion option in 2.* was `validate_success_only=true`, but this becomes `validate_success_only=false` in 3.*. For the smoothest possible upgrade, you should set it to `false` in your test suite before upgrading to 3.*.
 
+**Test schema coverage**  
+NOTE: Currently committee only supports schema coverage for openapi schemas, and only checks coverage on responses.
```
Comment (ota42y) [inline README.md:407]: Now we support tests using 'assert_schema_conform' or 'assert_response_schema_confirm' only.
(i.e. the user who use committee middleware in test )

So please write NOTICE to these user

File: test/data/openapi3/coverage.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    get:
+      description: get post
+      responses:
+        '200':
+          description: success
+          content:
+            application/json:
+              schema:
+                type: object
+        '404':
```
Comment (ota42y) [inline test/data/openapi3/coverage.yaml:19]: OpenAPI 3 support `default.
When status code wasn't matched defined, use `default` definithion.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#responses-object-example

So please implement and test it.

**Review by ota42y — CHANGES_REQUESTED** [review]
It's very good feature! 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 
But I add few comments, please check it.

File: README.md (hunk 1 above)
Comment (datbth) [reply README.md]: I have added more details to the docs

File: test/data/openapi3/coverage.yaml (hunk 2 above)
Comment (datbth) [reply test/data/openapi3/coverage.yaml:19]: done via https://github.com/interagent/committee/pull/297/commits/0712eedd3eb1e433dd2434b46912d6199cd7b1bb

**Comment by datbth** [comment]
Just made this commit https://github.com/interagent/committee/pull/297/commits/32600b24c4835e138a72b376f62885b8fbcf0bf5 to fix committee accepting requests even when they don't have matching prefix (in tests)

File: README.md [hunk 3]
```diff
@@ -403,6 +403,62 @@ end
...
 
+**Test schema coverage**  
+You can check how much of your API schema your tests have covered.  
+NOTICE: Currently committee only supports schema coverage for **openapi** schemas, and only checks coverage on responses, via `assert_response_schema_confirm` or `assert_schema_conform` methods.  
+Usage:
+1. Set schema_coverage option of `committee_options`
+2. Use `assert_response_schema_confirm` or `assert_schema_conform`
+3. Then use `SchemaCoverage#report` or `SchemaCoverage#report_flatten` to get coverage report
+
+Example:
```
Comment (ota42y) [inline README.md:414]: very nice!!!!!!!!!!!!!!!!! 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Review by ota42y — APPROVED** [review]
OK, I think this is very great changes!!!!!! Thanks!!!!!!!

**Comment by datbth** [comment]
hmm I have just realized I put the docs in the changes/migration section, should have put it in the upper big Test Assertion section instead :crying_cat_face:

**Pull request merged**